crypto/dh/dh_kdf.c

```diff
@@ -185,3 +190,4 @@ int DH_KDF_X9_42(unsigned char *out, size_t outlen,
     EVP_MD_CTX_cleanup(&mctx);
     return rv;
 }
+#endif
```

Comment:
@levitte I'll take a look at the dummy thing.

In the meantime, since the patches are owned by my company all I can do is raise the issue to legal again. I assume this CLA issue got lost in the noise.